localstack/services/cloudformation/models/route53.py

```diff
@@ -40,16 +40,36 @@ def param_change_batch(params, **kwargs):
             attrs = select_attributes(params, attr_names)
             alias_target = attrs.get("AliasTarget", {})
             alias_target["EvaluateTargetHealth"] = alias_target.get("EvaluateTargetHealth", False)
+            attrs["ResourceRecords"] = [{"Value": r} for r in attrs["ResourceRecords"]]
             return {
                 "Comment": params.get("Comment", ""),
                 "Changes": [{"Action": "CREATE", "ResourceRecordSet": attrs}],
             }
 
+        def hosted_zone_id_change_batch(params, **kwargs):
+            route53 = aws_stack.connect_to_service("route53")
+            hosted_zone_id = params.get("HostedZoneId")
+            if not hosted_zone_id:
+                hosted_zone_name = params.get("HostedZoneName")
+                # https://docs.aws.amazon.com/Route53/latest/APIReference/API_ChangeResourceRecordSets.html"
+                # "Specify either HostedZoneName or HostedZoneId, but not both. If you have multiple hosted zones with
+                # the same domain name, you must specify the hosted zone using HostedZoneId."
+                if not hosted_zone_name:
+                    raise Exception("Either HostedZoneId or HostedZoneName must be present.")
```

Comment:
Not happy about throwing Exception here, but also unsure which one I should pick here

Reply:
Do you mean the exception class or the wording of the message?

Reply:
Exception class

Reply:
Don't think we have anything special here right now to indicate these, so I'd say it's fine as is :+1: